src/locations/Field/Thumbnail.tsx

```diff
@@ -0,0 +1,220 @@
+
+  return (
+    <div ref={setNodeRef}>
+      <AssetCard
+        style={{
+          transform: CSS.Transform.toString(transform),
+          transition,
+          width: '100%',
+          maxWidth: '225px',
+        }}
+        dragHandleRender={() => (
+          <DragHandle 
+            as="button" 
+            style={{ alignSelf: 'stretch' }} 
+            label="Move card" 
+            {...attributes} 
+            {...listeners} 
+          />
+        )}
+        withDragHandle={!isDisabled}
+        src={url}
+        title={alt}
+        type="image"
+        icon={<img src={DEFAULT_INTEGRATION_PARAMETERS.logo} alt="" width={20} height={20} />}
+        size="small"
+        actions={[
+          <MenuItem 
+            key="view-on-imagekit" 
+            as="a" 
+            href={consoleUrl} 
+            target="_blank" 
+            onClick={() => window.open(consoleUrl, "_blank")}
+            style={{ fill: tokens.gray900 }}
+          >
+            <img src={DEFAULT_INTEGRATION_PARAMETERS.logo} alt="" width={15} height={15} /> View on ImageKit
+          </MenuItem>,
+          <MenuItem 
+            key="live-preview" 
+            as="a" 
+            href={asset.url} 
+            target="_blank" 
+            onClick={() => window.open(asset.url, "_blank")}
+            style={{ fill: tokens.gray900 }}
+          >
+            <ExternalLinkIcon /> Preview in New Tab
+          </MenuItem>,
+          <MenuItem key="remove" onClick={onDelete} isDisabled={isDisabled}>
+            <DeleteIcon /> Remove
+          </MenuItem>,
+          <MenuDivider key="divider" />,
+          <Menu.SectionTitle key="file-information-title">File information</Menu.SectionTitle>,
+          <MenuItem 
+            key="file-information" 
+            isDisabled 
+            style={{ opacity: 1 }}
+          >
+            <dl style={{
+              backgroundColor: tokens.gray100,
+              borderRadius: tokens.borderRadiusMedium,
+              padding: tokens.spacingXs,
+              width: '200px',
+              lineHeight: tokens.lineHeightS,
+              fontSize: tokens.fontSizeS,
+            }}>
+              <dt style={{
+                color: tokens.gray700,
+                marginRight: tokens.spacingXs,
+                paddingTop: tokens.spacing2Xs,
+                paddingBottom: tokens.spacing2Xs,
+                float: 'left',
+                clear: 'left',
+              }}>File path:</dt>
+              <dd style={{
+                marginLeft: 0,
+                color: tokens.gray900,
+                paddingTop: tokens.spacing2Xs,
+                paddingBottom: tokens.spacing2Xs,
+              }}>
+                <a href={folderUrl} target="_blank" rel="noopener noreferrer" style={{
+                  color: tokens.gray900,
+                  textDecoration: 'underline',
+                }}>
+                  {filePath}
+                </a>
+              </dd>
```

Comment:
Should we also add another field here that says imagekitId?

![image](https://github.com/user-attachments/assets/bc0dafa2-d376-47bb-abb9-71f135ea4f42)

Because EML can be opened from any account and imported from it, we don't have any locking mechanism right now that only this account is allowed to import into contentful

Reply:
just information

Reply:
done